docs/install/harvester-configuration.md

```diff
@@ -540,6 +540,21 @@ install:
   persistent_partition_size: 150Gi
 ```
 
+### `install.skipchecks`
+
+#### Definition
+
+The installer automatically checks if the hardware meets the [minimum requirements](/v1.4/install/requirements/#hardware-requirements) for production use. If any of the checks fail, installation is stopped, and warnings are printed to the system console and saved to `/var/log/console.log` in the installation environment.
```

Comment:
Should we link to the file relatively with `[minimum requirements](./requirements.md#hardware-requirements)`? I have the same suggestion for other files.

Reply:
Oh, that's a good idea, let me fix that (it'll be a PITA for version bumps in future otherwise)